src/components/DayOverview/DayOverview.tsx

```diff
@@ -23,19 +23,14 @@ export interface DayOverviewData {
 }
 
 export const getCurrentDate = (date: Date, hyphen: boolean = false): string => {
-    const formattedDate = date
-        .toLocaleString("en-GB", {
-            day: "2-digit",
-            month: "2-digit",
-            year: "numeric",
-        })
-        .split("/");
+    const formattedDate = date.toLocaleString("en-CA", {
+        year: "numeric",
+        month: "2-digit",
+        day: "2-digit",
+    });
 
-    const year = formattedDate[2];
-    const month = formattedDate[1];
-    const day = formattedDate[0];
-
-    return hyphen ? `${year}-${month}-${day}` : `${year}${month}${day}`;
+    const separator = hyphen ? "-" : "";
+    return formattedDate.replaceAll("-", separator);
```

Comment:
this 'does nothing' in one of the branches - because you have now changed it to `en-CA`, it's just replacing hyphens with hyphens, which is inefficient.

Maybe return the formatted date if hyphen is false, and formattedDate with hyphens replaced if hyphen is true? (ternary?)

Reply:
Done